packages/lodestar/src/network/network.ts

```diff
@@ -132,25 +137,24 @@ export class Libp2pNetwork extends (EventEmitter as {new (): NetworkEventEmitter
    * @param subnet the subnet calculated from committee index
    * @param inPeerIds peers already have this subnet
    */
-  private async connectToNewPeersBySubnet(subnet: number, inPeerIds: string[] = []): Promise<number> {
+  public async connectToNewPeersBySubnet(subnet: number, inPeerIds: string[] = []): Promise<number> {
     const discv5Peers = (await this.searchDiscv5Peers(subnet)) || [];
     const peerIds = discv5Peers.filter((peerId) => !inPeerIds.includes(peerId.toB58String()));
```

Comment:
yeah that's a good idea to me because  there are a lot of errors when we connect to peers from discv5, doing that will filter some bad peers that we disconnected in the past, will try that